themes/default/content/docs/pulumi-cloud/deployments/drift.md

```diff
@@ -66,22 +66,22 @@ You can integrate Drift notifications to Slack, MS Teams, and more using Pulumi
 
 ![Drift Webhook Events](../drift-webhook-events.png)
 
-Summary of Drift Events
+### Summary of Drift Detection events
```

Comment:
These are not just drift detection events, they are drift remediation as well, so this is actually wrong.

Reply:
gotcha; i'll put it back. thx